src/__forks__/createRequestError.js

```diff
@@ -0,0 +1,10 @@
+import formatRequestErrors from './formatRequestErrors';
+
+export default function createRequestError(request, requestType, payload) {
```

Comment:
I can, but the code is slightly different. Did you thing it is really necessary?